.github/workflows/ci.yml

```diff
@@ -108,6 +108,16 @@ jobs:
         lfs_sha: ${{ needs.cache_lfs.outputs.lfs_sha}}
         workflow_label: 'consumer'
 
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install pytest
```

Comment:
why would that be needed in the generic ci ?

Reply:
Because `ctest` invokes `pytest` now.